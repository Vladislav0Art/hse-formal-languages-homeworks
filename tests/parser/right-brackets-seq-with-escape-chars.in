start=$e\$cap\\e$
$e\$cap\\e$ -> '(' $e\$cap\\e$ ')' | NULL;