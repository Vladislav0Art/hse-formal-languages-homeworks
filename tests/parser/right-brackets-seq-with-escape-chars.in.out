-- Grammar -- 

Terminals: {')', '('}

Non-terminals: {'e$cap\\e'}

Start: 'e$cap\e'

Rules: {
	Bind {
	name: 'e$cap\e',
	expr: Alt(
			Combination(
				Terminal(value='('),
				NonTerminal(value='e$cap\\e'),
				Terminal(value=')'),
			),
			Null(),
		)
	}
}

Chomsky normal form: False

